pyomo/contrib/mindtpy/config_options.py

```diff
@@ -237,15 +156,177 @@ def _get_GDPopt_config():
         description="use fbbt to tighten the feasible region of the problem",
         domain=bool
     ))
+    CONFIG.declare("use_dual_bound", ConfigValue(
+        default=True,
+        description="add dual bound constraint to enforce the objective function should improve on the best found dual bound",
```

Comment:
The description is a little wrong. The objective function can never improve beyond the dual bound. Correct to say "add dual bound constraint to enforce the objective satisfies best-found dual bound"